controllers/airline_controller_test.go

```diff
@@ -58,9 +61,8 @@ func TestHandleAirlineById(t *testing.T) {
 
 func TestHandleCreateNewAirline(t *testing.T) {
 	beforeEachAirlineTest(t)
-	airline := models.Airline{
-		Name: "XYZAirline",
-	}
+	airline := factory.ConstructAirline()
+	airline = airline.SetName("XYZAirline")
```

Comment:
you can now extract the string literal into a local constant/variable and use it in both places instead of duplicating

Reply:
Fixed by extracting the string into a local variable.